packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -365,7 +365,7 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
 
     final Offset startOffset = _textPainter.getOffsetForCaret(
       TextPosition(offset: _selection.start, affinity: _selection.affinity),
-      Rect.zero,
+      _caretPrototype,
```

Comment:
This particular usage does not care what is passed in for the caretPrototype parameter. We pass in the _caretPrototype to not mess up the cached values and avoid redundant re-computation.